01-node-tutorial/answers/utils.js

```diff
@@ -0,0 +1,6 @@
+const value = "util";
+const utilFunction = (value) => {
+  return `From ${value}.js`;
+};
+
+module.exports = utilFunction(value);
```

Comment:
Small oversight here. You actually do not want to invoke the util Function. This will call the function on import which we do not want. Instead, try without the parentheses and argument

Reply:
Yeah I see my mistake, which I will fix.